focus-android/app/src/main/AndroidManifest.xml

```diff
@@ -180,11 +180,13 @@
         <service
             android:name=".downloads.DownloadService"
             android:exported="false"
-            android:foregroundServiceType="dataSync" />
+            android:foregroundServiceType="dataSync"
+            tools:ignore="ForegroundServicePermission" />
```

Comment:
This looks like a false positive because the required permission is in the merged manifest and not in the Focus manifest. An alternative would be adding the permissions explicitly.

Reply:
I guess it would be better for future references to have the permissions declared explicitly instead of having suppressions ( that would likely be fixed in future lint releases)